docs/src/references/c8y-configuration-management.md

```diff
@@ -162,4 +162,233 @@ Note that:
   for instance `tedge/configuration_change//etc/tedge/mosquitto-conf/c8y-bridge.conf`.
 * Since the type of configuration file is used as an MQTT topic name, the characters `#` and `+` cannot be used in a type name.
   If such a character is used in a type name (or in the path of a configuration file without explicit type),
-  then the whole plugin configuration `/etc/tedge/c8y/c8y-configuration-plugin.toml` is considered ill-formed.
\ No newline at end of file
+  then the whole plugin configuration `/etc/tedge/c8y/c8y-configuration-plugin.toml` is considered ill-formed.
+* Notifications for child-devices and their external devices are handled differently, as configuration files are fetched/pushed from/to external devices via HTTP file transfer. Thereby notifications are part of the file transfer contract (details see section [Fetching/Pushing configuration file](#fetchingpushing-configuration-file-fromto-the-external-device) below).
+
+## Configuration files for child devices
+
+To manage configuration files for child-devices the `c8y_configuration_plugin` aspects as below are relevant:
+
+* Associating configurations with cloud's child-device twin
+  * an external device declares its supported config-types to the plugin via MQTT, containing a child-id as reference to the child-device twin
+  * the plugin associates the reported config-type list from the external device to its cloud device twin matching the child-id
+  * the plugin stores the supported config-types of each child-device in a separate config-file
+  * the config-list files of each child-device is also managed by Configuration Management feature from C8Y
+* Consuming/Providing files from/to thin-edge via network
+  * any configuration file exchange to/from the external child device is handled by an HTTP based file transfer service provided by thin-edge
+
+## Managing Supported Configuration List of child devices
+
+To announce a child-devices configuration management capability child-device provisioning as described in [child-device reference documentation](child-devices.md#1-child-device-provisioning) is used.
+
+The `capability specific JSON object` for `c8y_configuration_plugin` contains all configurations the external device provides. Thereby each configuration appears with a `type`.
+
+The MQTT message is as below:
+
+```json
+{
+   "configurations": [
+     {
+       "type": "<config type 1>"
```

Comment:
Having only a configuration type sounds Cumulocity specific. While the generic `configurations` record suggest something more generic.

Reply:
With one of the latest spec changes there is here now mainly a `path` instead of a `type`. The `path` is the mandatory field, used as unique reference between the child-device and the plugin. The `type` can be optionally defined, or value `path` will be used for `type` as well.

So now we are here more decoupled from C8Y, and the behaviour is more similar to the one of configurations for the thin-edge device.